jenkins/jobs/macros-common.yaml

```diff
@@ -108,6 +118,8 @@
           sudo chmod -R 0777 ~/.ccache/
           #sudo chmod -R 0777 /home/jenkins/.ccache/
           pip install -r swift-uploader/requirements.txt
+          mkdir -pm 777 logs/domzero_logs
```

Comment:
We might need 777; it depends who needs to list files in the directory.  Note that the execute permission is needed for directories to list files within them.